utils/utils.js

```diff
@@ -0,0 +1,3 @@
+export function between(min, max) {
+  return Math.floor(Math.random() * (max - min) + min)
+}
```

Comment:
What do you think of moving this function to `utils/index.js`? This file is being used as "util" already, and although it's not very organized, I believe that it's worth it in order to keep the consistency.

I would also suggest renaming this function to a verb, due to consistency as well. `getRandomInt` looks good to me, as it's [used on MDN Math.random reference](https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/Math/random#gerando_um_n%C3%BAmero_inteiro_aleat%C3%B3rio_entre_dois_valores).

Reply:
I thought index.js was most related to the cypress, but I'm ok with this change :)
I agree with function name change too.
On it...

Reply:
Done